docs/modules/reference/pages/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -320,72 +344,91 @@ Note that you must include the `monitor` section for each service in your defini
+<monitor service="OpenNMS-Web-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <9>
 ----
-<1> Required *monitor* section.
+<1> The name of the page-sequence.
+<2> The relative URL to call in the request.
+<3> Host port number to use when connecting.
+<4> Set the virtual host field in the HTTP header.
+<5> Text to look for in the response body.
+<6> HTTP method (GET or POST)
+<7> Text to look for in the response body.
+<8> The failure message constructs the reason code.
+<9> Required *monitor* section.
 
 .Log in to `demo.opennms.org` without knowing username and password
 [source, xml]
 ----
 <service name="OpenNMS-Demo-Login" interval="300000" user-defined="true" status="on">
   <parameter key="page-sequence">
     <page-sequence>
-      <page path="/opennms"
-            port="80"
-            virtual-host="demo.opennms.org"
-            successMatch="(?s)User:.*<strong>(.*?)</strong>.*?Password:.*?<strong>(.*?)</strong>">
-        <session-variable name="username" match-group="1" />
-        <session-variable name="password" match-group="2" />
+      <page path="/opennms" <1>
+            port="80" <2>
+            virtual-host="demo.opennms.org" <3>
+            successMatch="(?s)User:.*<strong>(.*?)</strong>.*?Password:.*?<strong>(.*?)</strong>"> <4>
+        <session-variable name="username" match-group="1" /> <5>
+        <session-variable name="password" match-group="2" /> <6>
       </page>
-      <page path="/opennms/j_acegi_security_check"
-            port="80"
-            virtual-host="demo.opennms.org"
-            method="POST"
-            successMatch="Log out">"
+      <page path="/opennms/j_acegi_security_check" <1>
+            port="80" <2>
+            virtual-host="demo.opennms.org" <3>
+            method="POST" <7>
+            successMatch="Log out">" <4>
         <parameter key="j_username" value="${username}" />
         <parameter key="j_password" value="${password}" />
       </page>
-      <page path="/opennms/j_acegi_logout"
-            port="80"
-            virtual-host="demo.opennms.org"
-            successMatch="logged off" />
+      <page path="/opennms/j_acegi_logout" <1>
+            port="80" <2>
+            virtual-host="demo.opennms.org" <3>
+            successMatch="logged off" /> <4>
     </page-sequence>
   </parameter>
 </service>
 
-<monitor service="OpenNMS-Demo-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <1>
+<monitor service="OpenNMS-Demo-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <8>
 ----
-<1> Required *monitor* section.
+<1> The relative URL to call in the request.
+<2> Host port number to use when connecting.
+<3> Set the virtual host field in the HTTP header.
+<4> Text to look for in the response body.
+<5> Assign the value of a regex match group to a session variable
+<6> Assign the value of a regex match group to a session variable
```

Comment:
This seems to be a duplicate, but when I look at the code, it's for password. Is there a way we could indicate the difference? Maybe:

```suggestion
<6> Assign the value of a regex match group to a session variable. In this case, password.
```